Epsilon.Host.Frontend/src/views/DashboardView.vue

```diff
@@ -39,15 +34,24 @@ const store = useStore()
 onMounted(() => {
     api?.filter.accessibleStudentsList().then((r: HttpResponse<User[]>) => {
         store.commit("setUsers", r.data)
-        store.commit("setCurrentUser", store.state.users[0])
+        store.commit(
+            "setCurrentUser",
+            store.state.users.find(
+                (u: User) => u._id === import.meta.env.VITE_USER_ID
+            )
+        )
     })
     api?.learning
         .domainDetail("hbo-i-2018")
         .then((r) => store.commit("setDomain", r.data))
 
+    api?.learning
+        .domainDetail("pd-2020-bsc")
+        .then((r) => store.commit("setPersonalDevelopment", r.data))
+
     api?.learning
         .outcomesList({
-            studentId: "20592",
+            studentId: import.meta.env.VITE_USER_ID ?? "00000",
```

Comment:
What happens if studentId is set to 00000?

Reply:
00000 would be used, and Canvas will throw unauthorized error.